app/views/_includes/application-card.njk

```diff
@@ -8,7 +8,7 @@
       </a>
       <span class="app-application-card__caption">{{ a.id }}</span>
     </h3>
-    <p class="govuk-!-margin-bottom-1 govuk-!-font-size-16">{{ a.course }} – {{ a.studyMode | lower }} at {{ a.location | replace(' - training location','') }}</p>
+    <p class="govuk-!-margin-bottom-1 govuk-!-font-size-16">{{ a.course }} – {{ a.studyMode | lower }} at {{ a.location.name }}</p>
```

Comment:
Is this because all the location names are now lovely?

Reply:
Yes, that's right. We now have structured data for the course location.